core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -264,11 +267,37 @@ public void selectAudioTrack(PlayerAudioTrack audioTrack) {
     }
 
     @Override
+    public boolean hasAvailableSubtitles() {
+        NoPlayerLog.w("Subtitles not implemented for Android Media Player");
```

Comment:
I don't think it should throw. Both these players adhere to the `Player` interface and it feels like we'd be leaking knowledge of which player the client is actually using if we had a thrown exception here.

If the client already knows which media player they're using, maybe we shouldn't have them both adhering to the same interface?